content/pytorch/concepts/tensors/terms/tensor/view/view.md

```diff
@@ -0,0 +1,162 @@
+---
+Title: '.view()'
+Description: 'Returns a new tensor with the same data as the self tensor but of a different shape or type'
+Subjects:
+  - 'Data Science'
+  - 'Machine Learning'
+Tags:
+  - 'AI'
+  - 'Data Types'
+  - 'Deep Learning'
+  - 'Functions'
+  - 'Math operations'
+  - 'Data Science'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+---
+--------------
+
+# Tensor.view(*shape)
+Returns a new tensor with the same data as the self tensor but of a different shape.
```

Comment:
Please reword this, it is plagiarised: https://pytorch.org/docs/stable/tensors.html